### PR TITLE
Pelicanize the template

### PR DESCRIPTION
**Pull request by 1oglop1**
This PR aims to convert a static website template into pelican powered website.
So it's easy to publish new events and deploy via CI/CD.

### CODE REVIEW

File: blog-pelicanconf.py [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+# REVERSE_CATEGORY_ORDER = False
+# RSS_FEED_SUMMARY_ONLY = True
+
+ROBOTS = 'index, follow'
+
+# SLUGIFY_SOURCE = "title"
+# SLUG_REGEX_SUBSTITUTIONS = (
+#     [("[^\\w\\s-]", ""), ("(?u)\\A\\s*", ""), ("(?u)\\s*\\Z", ""), ("[-\\s]+", "-")],
+# )
+SOCIAL = (
```
Comment (gilgamezh) [inline blog-pelicanconf.py:225]: is this in use?

File: blog-pelicanconf.py [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+# IGNORE_FILES = ["ign/.*"]
+# INDEX_SAVE_AS = "index.html"
+# INTRASITE_LINK_REGEX = "[{|](?P<what>.*?)[|}]"
+JINJA_ENVIRONMENT = {
+    "extensions": ["jinja2.ext.i18n"],
+    "lstrip_blocks": True,
+    "trim_blocks": True,
+}
+# JINJA_FILTERS = {}
+LINKS = (
```
Comment (gilgamezh) [inline blog-pelicanconf.py:157]: should we update this links?

File: blog-pelicanconf.py [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+# TAG_SAVE_AS = "tag/{slug}.html"
+# TAG_URL = "tag/{slug}.html"
+# TEMPLATE_EXTENSIONS = [".html"]
+# TEMPLATE_PAGES = {}
+THEME = CUR_DIR.joinpath("themes", "jan-blog")
+# THEME_STATIC_DIR = "theme"
+# THEME_STATIC_PATHS = ["static"]
+# THEME_TEMPLATES_OVERRIDES = []
+TIMEZONE = "Europe/Amsterdam"
+TWITTER_USER_NAME = '1oglop1'
```
Comment (gilgamezh) [inline blog-pelicanconf.py:248]: @pyamsterdam

**Review by gilgamezh — APPROVED** [review]
thanks! really good idea.  :-) 

Which CI/CD should we use?

File: blog-pelicanconf.py (hunk 1 above)
Comment (1oglop1) [reply blog-pelicanconf.py:225]: whole file is not in use

**Comment by 1oglop1** [comment]
> thanks! really good idea. :-)
> 
> Which CI/CD should we use?

It's not ready for CI/CD yet, still the little things to make happen

**Pull request closed without merging**